### PR TITLE
remove docstring yaml parsing and openapi generation

### LINKED ISSUE #77: remove openapi* functions?
mb it worth to completely remove openapi* functions and stick to just yaml files. I found myself never using these functions anymore.

### PR DESCRIPTION
**Pull request by dselivanov**
fixes #77

### CODE REVIEW

File: R/Application.R [hunk 1]
```diff
@@ -325,43 +304,30 @@ Application = R6::R6Class(
...
       }
       self$logger$info("", context = list(endpoints = self$endpoints))
       return(invisible(self))
     },
-    #------------------------------------------------------------------------
-    add_openapi = function(path = "/openapi.yaml", openapi = openapi_create(),
-                           file_path = "openapi.yaml") {
+    add_openapi = function(path = "/openapi.yaml", file_path = "openapi.yaml") {
       checkmate::assert_string(file_path)
       file_path = path.expand(file_path)
```
Comment (artemklevtsov) [inline R/Application.R]: Must assert that the `file_path` exists and have yml extension.
```r
checkmate::assert_file_exists(file_path, extension = c("yaml", "yml"))
```
Also we must sure that `path` is string.

File: R/Application.R [hunk 2]
```diff
@@ -325,43 +304,30 @@ Application = R6::R6Class(
...
       # https://www.quora.com/What-is-the-correct-MIME-type-for-YAML-documents
       self$add_static(path = path, file_path = file_path, content_type = "application/x-yaml")
       return(invisible(file_path))
     },
-    #------------------------------------------------------------------------
-    add_swagger_ui = function(path = "/swagger",
-                              path_openapi = "/openapi.yaml",
-                              use_cdn = TRUE,
-                              path_swagger_assets = "/__swagger__/",
+    add_swagger_ui = function(path = "/swagger", path_openapi = "/openapi.yaml",
```
Comment (artemklevtsov) [inline R/Application.R]: Don't we need any check to OpenAPI yaml added before?

File: inst/examples/openapi/app.R [hunk 3]
```diff
@@ -65,11 +39,9 @@ app$add_get(
   path = "/fib",
   FUN = fib_handler
 )
-
 app$add_openapi(
   path = "/openapi.yaml",
-  openapi = openapi_create(),
-  file_path = tempfile(fileext = ".yaml")
+  file_path = "openapi.yaml"
```
Comment (artemklevtsov) [inline inst/examples/openapi/app.R]: Where is this file? I don't see it in the commit.

File: R/Application.R (hunk 2 above)
Comment (dselivanov) [reply R/Application.R]: in theory you can first add swagger-ui resource and then openapi resource

Comment (artemklevtsov) [reply R/Application.R]: I'm too worried about the user.

**Pull request merged**